packages/mdc-drawer/slidable/foundation.js

```diff
@@ -54,7 +54,7 @@ export class MDCSlidableDrawerFoundation extends MDCFoundation {
 
     this.inert_ = false;
 
-    this.drawerClickHandler_ = (evt) => evt.stopPropagation();
+    this.drawerClickHandler_ = () => {};
```

Comment:
I was actually just too lazy to mess around with the tests ;). But since it is basically "dead code", it's better to remove it. I added a new commit.